### DISCUSSION TITLE
Improve chapter navigation index

### DISCUSSION THREAD

**Discussion by dreaktr** [Tetrakern/fictioneer#58] [Ideas]
Improve the chapter navigation index by including sort by order feature. Not by date published or modified, but by order/reverse order as defined in the chapter's fcn_story edit post page. 

Kinda hard to select/navigate chapters when you have 100+ chapters. 

Also, and if the chapters have defining groups, maybe include the groups as toggleable(?) section which opens grouped chapters in navigation index? 

Something like this

![image](https://github.com/user-attachments/assets/26dfe5b5-a360-4d09-b5b0-e6c3636439da)

**Comment by Tetrakern** [comment]
I'm inclined to agree, but this won't be a priority. Honestly, I didn't think anyone even used this menu. It's expensive to render and who jumps around the story from a chapter, instead of going forward until you reach the end? Not once in my life have I ever used any of these menus. Well, we'll see.
